test/james.crypto.md5.tests.pas

```diff
@@ -76,8 +82,8 @@ procedure TMD5StreamTest.StreamFromMemory;
 initialization
   RegisterTestS(
     'Crypto', [
-      TMD5HashTest,
-      TMD5StreamTest
+      TMD5HashTest{$IFNDEF FPC}.Suite{$ENDIF},
```

Comment:
Yes, I agree. I'm not sure yet which will be the best solution, but I do think we need to address it.